src/V2/Records/ListRelatedQuery.php

```diff
@@ -0,0 +1,141 @@
+<?php
+
+namespace Zoho\Crm\V2\Records;
+
+use Zoho\Crm\Contracts\PaginatedQueryInterface;
+use Zoho\Crm\Contracts\ResponseTransformerInterface;
+use Zoho\Crm\Support\Helper;
+use Zoho\Crm\V2\Traits\HasPagination;
+
+class ListRelatedQuery extends AbstractQuery implements PaginatedQueryInterface
+{
+    use HasPagination;
+
+    /**
+     * @var string The record id
+     */
+    protected $recordId;
+
+    /**
+     * @var string The related list API name
+     */
+    protected $relatedList;
```

Comment:
I know that Zoho describes it as a "related list" but I don't think it is very meaningful. I would prefer to call it the "related module name" because that is what it is. Let's just call this property `$relatedModule`, and its description "The related module API name" please.
Also please update the related methods names.